### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by alolita**
Fixed organization links for TOC liaisons to TAG.

### CODE REVIEW

**Review by RichiH — APPROVED** [review]

**Review by kenfinnigan — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -74,25 +74,26 @@ If you want to propose new TAG activities or join in for existing ones, please t
 |------------------ | ------------------- |------------ |--------    | --------- | ---------------------------------- |
 | Bartłomiej Płotka | bwplotka@gmail.com  | @bwplotka   | [bwplotka] | [Red Hat] |  [Prometheus Team]; [Thanos Team]; [Other][bwplotka-other] |
```
Comment (bwplotka) [inline README.md:75]: While we are here 🙈 

```suggestion
| Bartłomiej Płotka | bwplotka@gmail.com  | @bwplotka   | [bwplotka] | [Google] |  [Prometheus Team]; [Thanos Team]; [Other][bwplotka-other] |
```

**Review by bwplotka — APPROVED** [review]
Thanks!

File: README.md [hunk 2]
```diff
@@ -74,25 +74,26 @@ If you want to propose new TAG activities or join in for existing ones, please t
...
 
-[Alibaba]:    https://www.alibaba.com
-[Amazon]:     https://aws.amazon.com
 [Apple]:      https://opensource.apple.com
 [Grafana]:    https://grafana.com
 [Microsoft]:  https://www.microsoft.com
 [Red Hat]:    https://redhat.com
```
Comment (bwplotka) [inline README.md:80]:
```suggestion
[Red Hat]:    https://redhat.com
[Google]:     https://about.google/
```

**Review by bwplotka — APPROVED** [review]
Thanks!

**Pull request merged**